tldw_Server_API/app/api/v1/endpoints/media.py

```
@@ -1429,10 +1432,38 @@
+                            "doi = EXCLUDED.doi, pmid = EXCLUDED.pmid, pmcid = EXCLUDED.pmcid, "
+                            "arxiv_id = EXCLUDED.arxiv_id, s2_paper_id = EXCLUDED.s2_paper_id"
+                        )
+                    else:
+                        ident_sql = (
+                            "INSERT OR REPLACE INTO DocumentVersionIdentifiers (dv_id, doi, pmid, pmcid, arxiv_id, s2_paper_id) "
+                            "VALUES (?, ?, ?, ?, ?, ?)"
+                        )
+                    db.execute_query(ident_sql, (dv_id, _doi, _pmid, _pmcid, _arxiv, _s2id), connection=conn)
+                except Exception:
+                    # Identifier table may not exist in older DBs; ignore
+                    pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)